compiler/hermes/include/hermes/core/Message.h

```diff
@@ -57,11 +58,17 @@ class Message final
   Message() = default;
 
 public:
-  void text(std::unique_ptr<MessageText> &&text) { _text = std::move(text); }
+  void text(std::unique_ptr<MessageText> &&text, SeverityCategory severity)
```

Comment:
You've removed existing `void text(std::unique_ptr<MessageText> &&text)` method.
Please leave existing method.